go/obscuronode/host/client_server.go

```diff
@@ -11,8 +11,9 @@ import (
 )
 
 const (
-	apiNamespaceObscuro = "obscuro"
-	apiVersion1         = "1.0"
+	apiNamespaceObscuro  = "obscuro"
+	apiNamespaceEthereum = "eth"
+	apiVersion1          = "1.0"
 )
 
 // An implementation of `host.ClientServer` that reuses the Geth `node` package for client communication.
```

Comment:
unrelated to this work. 
We should find a better name for "ClientServer". And I guess this pr is as good as any for it. 
Anything more explicit about what it actually is.